.appveyor.yml

```diff
@@ -0,0 +1,35 @@
+version: 2.x-{build}
+build: false
+clone_depth: 2
+clone_folder: c:\projects\sentry-php
+skip_branch_with_pr: true
```

Comment:
I've disabled builds on branch where there's an open PR. This way you can always see a build, even before opening the PR.